### PR TITLE
Change minScore label and add file upload for avatar space settings

### PR DESCRIPTION
**Pull request by samuveth**
I'm not sure if we really need a composable for the upload, let me know what you think.

Also updated docs https://app.gitbook.com/@bonustrack/s/snapshot/spaces/create

### CODE REVIEW

File: src/components/Upload.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <UiLoading v-if="loading" />
+  <label v-else class="file-select">
+    <input type="file" @change="handleFileChange" accept="image/*, video/mp4" />
```
Comment (bonustrack) [inline src/components/Upload.vue:4]: We should accept only image here
```suggestion
    <input type="file" @change="handleFileChange" accept="image/*" />
```

File: src/components/Upload.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    {{ $t('upload') }}
```
Comment (bonustrack) [inline src/components/Upload.vue:5]: It's better to use `<slot/>` to keep this component generic, later we will have the avatar displayed and upload button will be the avatar.

File: src/components/Token.vue [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 <script>
 import { formatBytes32String } from '@ethersproject/strings';
-
+import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';
```
Comment (bonustrack) [inline src/components/Token.vue:9]:
```suggestion
import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';

```

**Review by bonustrack — COMMENTED** [review]
Yes no need to be a composable, component works well! Added few suggestions

File: src/components/Token.vue (hunk 3 above)
Comment (samuveth) [reply src/components/Token.vue:9]: What does this do?

Comment (bonustrack) [reply src/components/Token.vue:9]: Just add a line between import and export

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**